crates/re_log_types/Cargo.toml

```diff
@@ -60,6 +60,7 @@ thiserror.workspace = true
 time = { workspace = true, features = ["formatting", "macros"] }
 typenum = "1.15"
 uuid = { version = "1.1", features = ["serde", "v4", "js"] }
+half.workspace = true
```

Comment:
sort! and maybe add a note about why it is here even though it is not a direct dependency